### DISCUSSION TITLE
Vertical line spacing between sprite columns - adjustable?

### DISCUSSION THREAD

**Discussion by andrew-davie** [stella-emu/stella#859] [Q&A] [answered]
The zookeper/moviecart spacing between adjacent sprites on alternating scanlines -- is this adjustable?
I just find the current visibility of the lines too annoying, and would love to be able to reduce the effect.

**Comment by andrew-davie** [comment]
I think one of the main issues I have is that although these spacings seem OK when Stella is fullscreen, they appear to me much more obvious when Stella is in a small window. Do they scale appropriately, I wonder?

**Comment by thrust26** [comment, marked as answer]
These are resulting from TV effects. Disable or reduce them.

**Reply by thrust26** [reply]
@andrew-davie Fine by you? Please mark as answer then.

**Reply by andrew-davie** [reply]
Yes, all good with effects disabled.
